Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM elixir:1.16.2-alpine AS build
+
+ARG MIX_ENV=prod
+
+RUN apk add --no-cache --update-cache \
+      git \
+      make
+
+RUN mix archive.install github hexpm/hex branch latest --force
+RUN /usr/local/bin/mix local.rebar --force
+
+RUN mkdir -p /opt/app
+ADD . /opt/app/
+
+WORKDIR /opt/app
+
+RUN mix deps.get --only $MIX_ENV
+RUN mix release --overwrite
+
+FROM alpine:3.19 as app
+
+RUN apk --no-cache add wireguard-tools iptables ip6tables inotify-tools libstdc++ libgcc u-boot-tools socat iproute2-ss agetty cgroup-tools
+RUN apk --no-cache add 'fwup~=1.10' \
+  --repository http://nl.alpinelinux.org/alpine/edge/community/
+
+# Use iptables masquerade NAT rule
+ENV IPTABLES_MASQ=1
+ENV PERIDIO_CONFIG_FILE=/etc/peridiod/peridio.json
+ENV UBOOT_ENV_SIZE=0x20000
```

Comment:
If this is only consumed at build time and not at run time of the container, then use `ARG` instead of `ENV`. It will "read" just like an ENV, but will be constrained to build time.